src/frontend/screens/Invites/InviteReceived.tsx

```diff
@@ -27,13 +26,13 @@ const m = defineMessages({
     id: 'screens.InviteReceived.declineInvite',
     defaultMessage: 'Decline Invite',
   },
-  joinProject: {
-    id: 'screens.InviteReceived.joinProject',
-    defaultMessage: 'Join {projectName}',
-  },
   invitedToJoin: {
     id: 'screens.InviteReceived.invitedToJoin',
-    defaultMessage: "You've been invited to join {projectName}",
+    defaultMessage: "You've been invited to join:",
```

Comment:
the designs have slightly different wording

```suggestion
    defaultMessage: "You've been invited to…",
```